src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -178,6 +189,40 @@ internal DecompressionMethods AutomaticDecompression
             }
         }
 
+        internal CookieUsePolicy CookieUsePolicy
```

Comment:
In fact, if you do that, you probably don't need the CookieUsePolicy enum at all.  Maybe I'm just misunderstanding what its purpose is?